src/styles/blocks/calendar.scss

```
@@ -33,12 +33,12 @@
     align-items: center;
     justify-content: center;
     background-color: #eee;
-    transition: color 0.5s, transform 0.5s;
+    transition: all 0.5s;
```

```suggestion
    transition: background-color 0.5s, transform 0.5s;
```

The previous solution was better. I suggest you add duration to variables.